CHANGELOG.md

```diff
@@ -4,6 +4,184 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.48.0] - 2024-05-29
+
+## 🚀 Features
+
+### Add support for studio trace id in selectors and document context entry ([Issue #3803](https://github.com/apollographql/router/issues/3803)), ([Issue #5172](https://github.com/apollographql/router/issues/5172))
+
+Add support for a new trace ID selector kind, the `apollo` trace ID, which represents the trace ID on [Apollo GraphOS Studio](https://studio.apollographql.com/). 
+
+An example configuration using `trace_id: apollo`:
+
+```yaml
+telemetry:
+  instrumentation:
+    spans:
+      router:
+        "studio.trace.id":
+            trace_id:: apollo
+```
+
+Also, router documentation has been updated with constants that are available in Rhai.
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/5189
+
+### Add ability for router to deal with query plans with contextual rewrites ([PR #5097](https://github.com/apollographql/router/pull/5097))
+
+Adds the ability for the router to execute query plans with context rewrites on them. These are generated by the @fromContext directive and are used to map a Value in the collected data JSON onto a variable which will in turn be used as an argument to a field resolver.
+
+⚠️ This ships with a new version of federation, 2.8.0, which means distributed caches will be repopulated.
```

Comment:
@clenfest - can we add a link to the docs here?

Reply:
I cannot find it in the router docs nor in the [PR](https://github.com/apollographql/router/commit/5a0826e6f22568bf61b99447501e2711bbf36edc) that introduced it. Could we add some docs for this feature before releasing it?

Reply:
@chandrikas the docs PR is in the federation repo https://github.com/apollographql/federation/pull/2975